src/plugins/auto/plugin.cpp

```diff
@@ -887,12 +884,17 @@ std::string MultiDeviceInferencePlugin::GetDeviceList(const std::map<std::string
                 updateDeviceVec(iter);
             }
             for (auto&& device : deviceVec) {
-                allDevices += device;
-                allDevices += ((device == deviceVec[deviceVec.size()-1]) ? "" : ",");
+                if (!_pluginConfig.isSupportedDevice(device))
+                    continue;
```

Comment:
I'm thinking, how can you judge if device is supported when in future we remove the white list(which was originally used for secondary property checking?)

Reply:
Will check if device is in blacklist in future, currently is checking if device is in whitelist.